packages/react-tokens/scripts/generateTokens.mjs

```diff
@@ -10,7 +10,7 @@ const version = 'v6';
 
 // Helpers
 const formatCustomPropertyName = (key) =>
-  key.replace(`--pf-${version}-`, '').replace('--pf-t--', '').replace(/-+/g, '_');
+  key.replace(`--pf-${version}-`, '').replace('--pf-t--', 't_').replace(/-+/g, '_');
```

Comment:
How does prefixing with "t_" resolve the issue with multiple exports? 

I'm also curious why the version has been removed?

Reply:
I built this branch locally and still see duplicates. The only difference now is one file is prefixed with "t_".

For example:
packages/react-tokens/dist/esm/chart_global_label_Margin.js
packages/react-tokens/dist/esm/t_chart_global_label_margin.js

Reply:
@dlabrecq the `packages/react-tokens/dist/esm/chart_global_label_Margin.js` is the variable.
the `packages/react-tokens/dist/esm/t_chart_global_label_margin.js` is the token.

@mcoker can add more context from a css perspective as to why they are named the same.

Reply:
Hey @dlabrecq. The original bug is because the PF CSS has variables named very similarly like these two - `--pf-v6-chart-global--label--Margin` and `--pf-t--chart--global--label--margin`. The react-tokens package parses the PF CSS, finds all of our variables, and creates a react-token for each one with a name based based off of what the CSS variable name is. Most of the PF CSS vars fit into 4 categories:
* Components - `--pf-v6-c-[component]` creates a react token like `c_[component]`
* Layouts - `--pf-v6-l-[layout]`, creates `l_[layout]`
* Charts - `--pf-v6-chart-[color/global/axis/bar/etc]`, creates `chart_[color/global/axis/bar/etc]`
* Tokens, aka global vars - `--pf-t--[color/global/chart/etc]`, creates `[color/global/chart/etc]_`

So for the vars reference above, `--pf-v6-chart-global--label--Margin` and `--pf-t--chart--global--label--margin`, the resulting tokens are `chart_global_label_Margin` and `chart_global_label_margin`. There are a bunch of ways we could address that, but since the CSS token vars (`--pf-t--[...]`) are always prefixed with `-t--`, prefixing the react token filename with `t_[...]` solves this problem and follows the format of other prefixed vars/tokens in PF (`--pf-v6-c` and `--pf-v6-l`).

Reply:
Thanks for the explanation. I didn't realize we wanted to expose both tokens and variables in this package